core/src/main/resources/org/apache/spark/ui/static/timeline-view.css

```diff
@@ -223,8 +223,8 @@ rect.getting-result-time-proportion {
 }
 
 .vis-timeline .vis-item.executor.vis-selected {
-  background-color: #A2FCC0;
-  border-color: #36F572;
+  background-color: #A0DFFF;
+  border-color: #3EC0FF;
   z-index: 2;
 }
```

Comment:
How about just removing this directive rather than modifying the color settings?
I've tried removing it and seems to works fine.